fiftyone/core/collections.py

```diff
@@ -9528,6 +9528,13 @@ def get_index_information(self, include_stats=False):
                 if key in sample_info:
                     sample_info[key]["in_progress"] = True
 
+            for d in self._dataset._sample_collection.aggregate(
```

Comment:
Wondering if it would be useful to pull this out and make this a function that accepts the collection name as an arg to allow for creating something like an admin dashboard to manage indexes across datasets without needing to load each dataset

Reply:
The use case makes sense 👍  I'd suggest we consider whether a potential refactor adds clarity at that time